src/index.spec.ts

```diff
@@ -4,9 +4,149 @@ import {
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
+/*test('Testing time!', () => {
   console.log('welcome');
+});*/
+
+describe('isInteger()',() => {
+  it("should return true if value is a number", () => {
+    expect(isInteger(1)).toBe(true);
+    expect(isInteger(-1)).toBe(true);
+    expect(isInteger(0)).toBe(true);
+  });
+  it('should return false if value is a string', () => {
+    expect(isInteger('test')).toBe(false);
+    expect(isInteger('111')).toBe(false);
+    expect(isInteger('-123')).toBe(false);
+  });
+});
```

Comment:
You should give a look to [jest-in-case](https://www.npmjs.com/package/jest-in-case). Basically, it allows us to write groups of similar test cases in a practical way.